doc/nrf/libraries/networking/nrf_cloud_pgps.rst

```diff
@@ -31,7 +31,7 @@ However, P-GPS should not be used for general use cases that already work with :
    When using two-week ephemeris prediction sets, the TTFF towards the end of the second week will increase due to the accumulated errors in the predictions and the decrease in the number of satellite ephemerides in the later prediction periods.
 
 P-GPS requires a cloud connection approximately once a week, depending on the configuration settings.
-A-GPS requires a cloud connection each time.
+A-GPS requires a cloud connection as often as once every two hours.
```

Comment:
```suggestion
A-GPS requires a cloud connection once every two hours.
```

Reply:
I said "as often as" because this is only the case if the application requires a position fix at least once every 2 hours.

For example, if the application only needs a fix once a day, A-GPS data will only be downloaded for that one fix, not for the 11 other 2 hour periods during which the application did not try to get a fix.

Reply:
Still, it sounds a bit weird. How about explaining what you just said. Something like "A-GPS requires a cloud connection as often as your application requires a position fix."

Reply:
OK, I tried again.